project/src/services/api.ts

```diff
@@ -0,0 +1,13 @@
+import axios, {AxiosInstance} from 'axios';
+
+const BACKEND_URL = 'https://10.react.pages.academy/six-cities';
+const REQUEST_TIMEOUT = 5000;
+
+export const createAPI = (): AxiosInstance => {
+  const api = axios.create({
+    baseURL: BACKEND_URL,
+    timeout: REQUEST_TIMEOUT,
+  });
+
+  return api;
+};
```

Comment:
пока бы я написал в одну строчку все таки. Сразу много букв экономится

```suggestion
export const createAPI = () => axios.create({
    baseURL: BACKEND_URL,
    timeout: REQUEST_TIMEOUT,
  });


```